beetsplug/bpd/__init__.py

```diff
@@ -874,7 +1088,11 @@ def play_finished(self):
         """A callback invoked every time our player finishes a
         track.
         """
+        if not self.ctrl_sock:
+            self.ctrl_sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            self.ctrl_sock.connect((self.ctrl_host, self.ctrl_port))
         self.cmd_next(None)
+        self.ctrl_sock.sendall(u'play_finished\n'.encode('utf-8'))
```

Comment:
It might be nice to wrap up the `ctrl_sock` setup and the `sendall` call into a new `ctrl_send` method that conveniently does both.